### DISCUSSION TITLE
Use tween.dynamic() to make a div follow the mouse smoothly ?

### DISCUSSION THREAD

**Discussion by optimus007** [tweenjs/tween.js#653] [Q&A]
tween.js fiddle https://jsfiddle.net/sdwyv249/7/

the example i want to emulate https://codepen.io/GreenSock/pen/WNNNBpo

the manual way would have been to make the div follow the mouse with a factor + timeDelta in the request animation frame loop

was wondering if it was possible to do the same with the new `dynamic` somehow ?

**Comment by trusktr** [comment]
@optimus007 Hello, sorry for such a late reply. I see both examples, but I'm not ure what you'd want to do differently. Do you still need help or did you figure it out already?

**Reply by optimus007** [reply]
Ohh

I think i used pmndrs/maath's "damp" functions to get similar result

But the question is

Can I update the tween end value dynamically while the tween is running and have the animation transition to it smoothly without stuttering/jumps ?

**Reply by trusktr** [reply]
I thought that's what you did with the `.dynamic(true)` option in your example. Do you mean something different?
